.nvmrc

```diff
@@ -0,0 +1 @@
+8.6
```

Comment:
I see what you did there

![ISWYDT](https://media1.giphy.com/media/12ZDIx1Mw1cXVm/giphy.gif)